libs/community/langchain_community/document_loaders/mongodb.py

```diff
@@ -20,6 +23,10 @@ def __init__(
         *,
         filter_criteria: Optional[Dict] = None,
         field_names: Optional[Sequence[str]] = None,
+        cursor_builder: Optional[Callable] = None,
+        page_content_mapper: Optional[Callable[..., str]] = None,
+        metadata_mapper: Optional[Callable[..., Dict[str, Any]]] = None,
+        enable_total_count_check: bool = True,
```

Comment:
could you include docstring for this? There's an neighboring PR that's [coming down the pipe](https://github.com/langchain-ai/langchain/pull/23376/files) that would also benefit from getting the docstring info.